source/documentation/appendix-docs/conda.md

```diff
@@ -1,89 +1,73 @@
-# Conda Package Management
+# Package management
 
-## Unified package management for both R and Python
+## Introduction
 
-### Faster builds
+A key element of ensuring analysis is reproducible is maintaining a record of the versions of packages used to produce said analysis. There are a number of package management tools available for both R and Python. Within the MoJ we recommend primarily using [Conda](#Conda) for both languages.
 
-CRAN mirrors do not provide linux compiled binaries
-for packages. This means a long wait when doing `install.packages` both in an R
-Studio session and when running a Docker build for a `shiny` application.
+## R
 
-### Cross Language
+### Packrat
```

Comment:
I think that Conda should come first here because it's the one we recommend - that way people who are time-poor will come to it straight away. You can then add the other alternatives in a section called something like "alternatives to Conda".

Reply:
Agreed. Fixed